R/foodweb.R

```diff
@@ -134,7 +140,7 @@
 #' \sQuote{name}.
 #' @references See them above.
 #' @source See references for the individual webs above. The data itself
-#'   was downloaded from
-#'   <http://vlado.fmf.uni-lj.si/pub/networks/data/bio/foodweb/foodweb.htm>.
+#'   was downloaded from the then webpage of the Pajek software
+#'  (link now broken).
```

Comment:
Please do not write "link now broken" in the documentation.

In this case, Google still finds the page and there is only a timeout. This suggests that the server is only down temporarily and will come back.

When this is not the case, it is best to get the last good snapshot of the page from archive.org. To do this, one needs the original link, another argument not to remove it.

Reply:
in my case the webpage never loaded. Do you want to link to the archive.dot snapshots for broken links then?

Reply:
I can't find a snapshot for this page on the wayback machine.

Reply:
In this case I think we should wait for a couple of days and see if comes back online (or just email Vlado and ask).

That said, this page _is_ present on the Wayback Machine and the datasets are downloadable:

https://web.archive.org/web/20230930092030/http://vlado.fmf.uni-lj.si/pub/networks/data/bio/foodweb/foodweb.htm

Reply:
Do you mind emailing Vlado? It'd be nice to link to an https not http page. :smile_cat:

Reply:
Friendly reminder :grin: